src/components/Table/GridTable.tsx

```diff
@@ -712,6 +752,16 @@ export function calcColumnSizes(columns: GridColumn<any>[], firstLastColumnWidth
 
   // This is our "fake but for some reason it lines up better" fr calc
   function fr(myFr: number): string {
+    // If the tableWidth, then return a pixel value
+    if (tableWidth) {
+      const widthBasis = tableWidth > tableMinWidthPx ? tableWidth : tableMinWidthPx;
+      // When the tableWidth is defined, then we need to account for the `firstLastColumnWidth`s.
+      return `(${
+        (widthBasis - (claimedPercentages / 100) * widthBasis - claimedPixels - (firstLastColumnWidth ?? 0) * 2) *
+        (myFr / totalFr)
+      }px)`;
+    }
```

Comment:
This was mostly stolen from SchedulesV2. Added in handling for "claimedPercentages" and `firstLastColumnWidth`